components/01-visual-styling/02-button/01-text-link/text-link.scss

```diff
@@ -11,7 +11,7 @@
     color: $white;
 }
 
-.blue-bg h2 {
+a.blue-bg h2 {
```

Comment:
It looks like the text isn't showing up on the blue banner:
https://utsa-asc.github.io/college-dls/431-dls-full-width-content-callout-no-image-no-cta/components/detail/text-link.html

Reply:
The issue stated make default white.

Reply:
I'll just add a gray variant as well. Problem between mockup examples and stated issue requirements solved!

Reply:
Sounds good! I wasn't sure if I missed something with the conversation.